src/scanner.rs

```diff
@@ -161,8 +162,30 @@ impl BundleRepo {
                 .string_by_key(GIT_REMOTE_ORIGIN_URL_CONFIG)
                 .map(|s| s.to_string());
             let mut git_head = git_repo.head()?;
+
+            let mut git_head_branch = git_head.referent_name().map(|s| s.as_bstr().to_string());
+            if git_head_branch.is_none() {
+                for r in git_repo.references()?.remote_branches()? {
+                    match r {
+                        Ok(r) => {
+                            let target = r.target();
+                            let id = target.try_id();
+                            if id.is_some()
+                                && git_head.id().is_some()
+                                && id.unwrap().to_string() == git_head.id().unwrap().to_string()
+                            {
+                                git_head_branch =
+                                    Some(r.name().to_path().to_str().unwrap().to_string());
+                                break;
+                            };
+                        }
+                        Err(e) => {
+                            log::warn!("unable to match reference {:?}", e);
```

Comment:
Why do we need to warn?

Reply:
We can lower it to info. I think right now we want to be verbose on logging to help with debugging.